packages/errors/index.ts

```diff
@@ -36,10 +36,19 @@ class MongoshWarning extends Error {
   }
 }
 
+class MongoshCommandFailed extends Error {
+  constructor(msg) {
+    super(msg);
+    this.name = 'MongoshCommandFailed';
+    this.message = `Command ${msg} returned ok: 0. To see the raw results of the command, use 'runCommand' instead.`;
```

Comment:
should we very explicitly say `db.runCommand`? Or do you think that would be redundant? 
```suggestion
    this.message = `Command ${msg} returned ok: 0. To see the raw results of the command, use 'db.runCommand()' instead.`;
```

Reply:
There is both db.runCommand and collection.runCommand, which do the same thing under the hood. But there are collection methods that will throw this error, so I figured it's better to just name the method as opposed to both classes and the method.

Reply:
ah, fair!